moveit_ros/planning_interface/CMakeLists.txt

```diff
@@ -8,88 +8,89 @@ if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
+#find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
 
 find_package(Boost REQUIRED)
-if(Boost_VERSION LESS 106700)
-  set(BOOST_PYTHON_COMPONENT python)
-elseif()
-  set(BOOST_PYTHON_COMPONENT python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
-endif()
+#if(Boost_VERSION LESS 106700)
+#  set(BOOST_PYTHON_COMPONENT python)
+#elseif()
+#  set(BOOST_PYTHON_COMPONENT python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
+#endif()
 
 find_package(Boost REQUIRED COMPONENTS
   date_time
   filesystem
   program_options
-  ${BOOST_PYTHON_COMPONENT}
+  #${BOOST_PYTHON_COMPONENT}
   system
   thread
 )
 find_package(Eigen3 REQUIRED)
-find_package(eigenpy REQUIRED)
+#find_package(eigenpy REQUIRED)
 
 set(THIS_PACKAGE_INCLUDE_DIRS
-  py_bindings_tools/include
-  common_planning_interface_objects/include
-  planning_scene_interface/include
-  move_group_interface/include
+  #py_bindings_tools/include
+  #common_planning_interface_objects/include
+  #planning_scene_interface/include
+  #move_group_interface/include
   moveit_cpp/include
 )
 
-catkin_python_setup()
-
-catkin_package(
-  LIBRARIES
-    moveit_common_planning_interface_objects
-    moveit_planning_scene_interface
-    moveit_move_group_interface
-    moveit_cpp
-  INCLUDE_DIRS
-    ${THIS_PACKAGE_INCLUDE_DIRS}
-  CATKIN_DEPENDS
-    actionlib
-    moveit_ros_planning
-    moveit_ros_warehouse
-    moveit_ros_manipulation
-    moveit_ros_move_group
-    geometry_msgs
-    moveit_msgs
-    tf2_geometry_msgs
-  DEPENDS
-    EIGEN3
+set(THIS_PACKAGE_LIBRARIES
+  #moveit_common_planning_interface_objects
+  #moveit_planning_scene_interface
+  #moveit_move_group_interface
+  moveit_cpp
 )
 
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+  rclcpp_action
+  moveit_ros_planning
+  #moveit_ros_warehouse
+  #moveit_ros_manipulation
+  #moveit_ros_move_group
+  geometry_msgs
+  moveit_msgs
+  tf2_geometry_msgs
+)
+
+#catkin_python_setup()
+
 include_directories(${THIS_PACKAGE_INCLUDE_DIRS}
-                    ${Boost_INCLUDE_DIRS}
-                    ${catkin_INCLUDE_DIRS})
+                    ${rclcpp_INCLUDE_DIRS}
+                    ${Boost_INCLUDE_DIRS})
 
 include_directories(SYSTEM
-                    ${EIGEN3_INCLUDE_DIRS}
-                    ${eigenpy_INCLUDE_DIRS}
-                    ${PYTHON_INCLUDE_DIRS})
-
-add_subdirectory(py_bindings_tools)
-add_subdirectory(common_planning_interface_objects)
-add_subdirectory(planning_scene_interface)
-add_subdirectory(move_group_interface)
-add_subdirectory(robot_interface)
-add_subdirectory(test)
+                    ${EIGEN3_INCLUDE_DIRS})
+		    #${eigenpy_INCLUDE_DIRS}
+		    #${PYTHON_INCLUDE_DIRS})
+
+#add_subdirectory(py_bindings_tools)
+#add_subdirectory(common_planning_interface_objects)
+#add_subdirectory(planning_scene_interface)
+#add_subdirectory(move_group_interface)
+#add_subdirectory(robot_interface)
+#add_subdirectory(test)
 add_subdirectory(moveit_cpp)
+
+ament_export_include_directories(include)
```

Comment:
Does this directory exist at `moveit_ros/planning_interface`?

Reply:
No, it exports the shared include directory

Reply:
It's the install directory. See 
https://github.com/ament/ament_cmake/blob/master/ament_cmake_export_include_directories/cmake/ament_export_include_directories.cmake#L37